.github/workflows/build-release-binary.yml

```diff
@@ -0,0 +1,129 @@
+name: "Build release binary"
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_binaries:
+    name: Build release binary
+    strategy:
+      matrix:
+        include:
+          - bin: taker
+            target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+            archive_ext: tar
+          - bin: taker
+            target: armv7-unknown-linux-gnueabihf
+            os: ubuntu-latest
+            archive_ext: tar
+          - bin: taker
+            target: x86_64-apple-darwin
+            os: macos-latest
+            archive_ext: tar
```

Comment:
on macos `zip` is probably more convenient choice, as it automatically unpacked by Safari when extracted

Reply:
Do you mind sending a follow-up patch if you care enough? :D

Testing Github workflows is a pain ...

Reply:
I'll look into it at some point, when all the MVP tasks are done :)